backport-changelog/6.9/9912.md

```diff
@@ -0,0 +1,3 @@
+https://github.com/WordPress/wordpress-develop/pull/9912
+
+* https://github.com/WordPress/gutenberg/pull/71660
```

Comment:
Need to remove this file. See https://github.com/WordPress/wordpress-develop/pull/9912#issuecomment-3341342908